demos/browser/webpack.config.js

```diff
@@ -40,11 +40,16 @@ for (const stage of ['a', 'b', 'g', '']) {
   csp['script-src-elem'] += host;
 }
 
-// 2. Access to jsdelivr for TensorFlow for background blur.
+// 2. Access to googleapis for the Segmentation filter 
+csp['connect-src'] += ' https://storage.googleapis.com';
+csp['script-src'] += ' https://storage.googleapis.com';
+csp['script-src-elem'] += ' https://storage.googleapis.com';
```

Comment:
I would be surprised if it needed `script-src-elem`, and even a little bit surprised if it needs `script-src`.